test/test/compound.test.js

```diff
@@ -0,0 +1,274 @@
+/**
+ * @format
```

Comment:
@yaronvel 
please notice 
only files with this annotation will be formatted 
so we can add this code style enforcement gradually